lib/lita/handlers/artifactory.rb

```diff
@@ -126,6 +126,11 @@ def promote(response)
           EOH
         end
 
+        # This event allows other lita utilities to act when projects are promoted.
+        robot.trigger(:artifactory_artifact_promoted, product: project, version: version)
```

Comment:
Where would I find those though? The target channel is probably `PROMOTION_CHANNEL` but I'm not seeing a place for source channel?